### ISSUE TITLE
filename like this [xxxxxx.vtt?language=en] not works

### ISSUE THREAD

**Issue by msongz** [whyboris/Simplest-File-Renamer#8]
will rename to xxxxx.vtt.[ext]?language=en

**Comment by whyboris** [comment]
Sorry, could you clarify -- the originl file name is `xxxxxxx.vtt` or something different?

**Comment by msongz** [comment]
yeah,i want to rename the file from `xxxxxx.vtt?language=en` to `xxxxxx.srt`
but it fail

**Comment by whyboris** [comment]
Thank you for clarifying!

The way I detect extensions is `path.parse(file);`

https://github.com/whyboris/Simplest-File-Renamer/blob/master/src/app/home/home.component.ts#L247

I'm unsure what's going wrong here. I'm currently more-eager to work on *Video Hub App*:
https://github.com/whyboris/Video-Hub-App

Pull requests are welcome for fixing this edge case 🤝

**Comment by whyboris** [comment]
It looks like `?` is not allowed in a file name (at least on _Windows_)

I've updated the code https://github.com/whyboris/Simplest-File-Renamer/commit/b40ebe9712ecc1030144b315e88c94cb4050a3a8 preventing all (Windows) forbidden characters - showing users a distinct message for the error they committed 👍 :trollface:👍

I'm not intending, at least at this time, to allow users to change the file extension within this app 👌 

Closing this _Issue_ - no renaming of extensions 🤝
